web/src/Sidebar.tsx

```diff
@@ -76,21 +83,26 @@ class Sidebar extends PureComponent<SidebarProps> {
         analyticsKey = "ui.interactions.errors"
         link += "/errors"
       }
-      let classes = `resLink resLink--${item.status}`
+
+      let formatter = buildFormatter(enStrings)
+      let hasBuilt = !isZeroTime(item.lastDeployTime)
+      let willBuild = !isZeroTime(item.pendingBuildSince)
```

Comment:
Actually, we should show this spinner when we're actively building a resource, not (just?) when there's a pending build.

Reply:
i think so yeah!